<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P2P</class>
 <widget class="QDialog" name="P2P">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	
	
	background-color: rgb(85, 170, 255);
	
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;等线&quot;;
	color: rgb(0, 0, 0);
 border-style: outset;
     border-width: 1px;
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	
	background-color: rgb(137, 194, 255);
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	
	
	
	background-color: rgb(162, 212, 255);
                   border-color:rgba(255,255,255,200);
                   
	color: rgb(0, 0, 0);
                   }</string>
   </property>
   <property name="text">
    <string>发送信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sendfile">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;等线&quot;;
	color: rgb(0, 0, 0);
 border-style: outset;
     border-width: 1px;
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	
	background-color: rgb(137, 194, 255);
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	
	
	
	background-color: rgb(162, 212, 255);
                   border-color:rgba(255,255,255,200);
                   
	color: rgb(0, 0, 0);
                   }</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_write">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-style: outset;
     border-width: 1px;
     border-radius: 4px;
border-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_show">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-style: outset;
     border-width: 1px;
     border-radius: 4px;
border-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
font: 7pt &quot;幼圆&quot;;
color:rgb(108, 108, 108);
 border-style: outset;
    
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	background-color: rgb(255, 255, 255);
border-top:0px solid #e8f3f9;
background:  transparent;
	image: url(:/icon/连接.png);
}
QPushButton:pressed{
	color: rgb(255, 255, 255);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	background-color:  rgb(255, 255, 255);
                   border-color: rgb(255, 255, 255);
                   
	border-top:0px solid #e8f3f9;
background:  transparent;
	
	color: rgb(11, 11, 11);
                   }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Chat </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/tubiao.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	background-color: rgb(137, 194, 255);
}
</string>
   </property>
   <item>
    <property name="text">
     <string>阳光男孩</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大树</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>老李</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>爱情1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小丑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>男神1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小帅哥</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>男神2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>男神3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>爱情2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>爱情3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>221</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
